stdlib/public/core/CollectionOfOne.swift

```diff
@@ -22,6 +22,7 @@
 ///     let b = a + CollectionOfOne(toAdd)
 ///     // b == [1, 2, 3, 4, 100]
 @frozen // trivial-implementation
+@_addressableForDependencies
```

Comment:
Does this have ABI implications?

Reply:
It does, specifically for the new case of functions returning non-escapable values. For existing functions returning escapable values it changes nothing. @jckarter knows more.